hknweb/candidate/urls.py

```diff
@@ -8,9 +8,10 @@
     # candidate end of officer challenge requests
     path('candreq', views.CandRequestView.as_view(), name='candrequests'),
     # officer end of officer challenge requests
-    # re_path(r'^challengeconfirm/(?P<pk>\d+)/$', views.OfficerConfirmView.as_view(), name='challengeconfirm'),
+    path('offreq', views.OffRequestView.as_view(), name='offrequests'),
     re_path(r'^challengeconfirm/(?P<pk>\d+)/$', views.officer_confirm_view, name='challengeconfirm'),
```

Comment:
This does not have to be regex:
```py
path('detail/<pk:int>/', ...)
```
I'm pretty sure this (or something similar) will do the job equally well.